connectathon-scenarios-2019-09/examples/scenario-01-outbound-request.json

```diff
@@ -0,0 +1,40 @@
+{
+  "hookInstance": "d1577c69-dfbe-44ad-ba6d-3e05e953b2ea",
+  "fhirServer": "http://hooks.smarthealthit.org",
+  "context": {
+    "userId": "Practitioner/123",
+    "patientId": "MRI-59879846",
+    "encounterId": "89284",
+    "selections": [
+      "ServiceRequest/example-MRI-59879846"
```

Comment:
Let's make sure the casing is consistent for the id between the selections and the `draftOrders` Bundle

Reply:
Oh, thanks -- yeah, that's an error.